src/plotman/csv_exporter.py

```diff
@@ -0,0 +1,115 @@
+import csv
+import sys
+from dateutil.parser import parse as parse_date
+from plotman.log_parser import PlotLogParser
+from plotman.plotinfo import PlotInfo
+
+def export(logfilenames, save_to = None):
+    if save_to is None:
+        send_to_stdout(logfilenames)
+    else:
+        save_to_file(logfilenames, save_to)
+
+def save_to_file(logfilenames, filename: str):
+    with open(filename, 'w') as file:
```

Comment:
https://docs.python.org/3/library/csv.html#examples

If I remember correctly, this relates to quoted fields with newlines in them.  Unlikely, but best to just always open files as the CSV modules suggests.  (detail, py2 is different...)

```suggestion
    with open(filename, 'w', newline='') as file:
```